docs/request_metadata.md

```diff
@@ -0,0 +1,70 @@
+# Logging request metadata 
+
+In Rails environment, every event is enhanced with the request metadata provided by `rack` server. This can help with debugging and building an audit log from events for the future use.
+
+## Defaults
+
+With default configuration metadata is enhanced by:
+
+* `:remote_ip` - IP of the HTTP client which issued the request.
+* `:request_id` - An unique ID of the request.
+
+This metadata is included **only for published events**. So creating a new event instance by hand won't add metadata to it:
+
+```ruby
+event = MyEvent.new(event_data)
+event.metadata # nil, unless you provided your own data called 'metadata'.
```

Comment:
Great! I just quickly scanned the code so I just missed it. Would you rephrase this paragraph to be correct?